src/components/OrganizationScreen/OrganizationScreen.test.tsx

```diff
@@ -81,15 +81,13 @@ describe('Testing OrganizationScreen', () => {
     fireEvent.click(closeButton);
 
     // Check for contract class after closing
-    expect(screen.getByTestId('mainpageright')).toHaveClass('_expand_ccl5z_8');
+    expect(screen.getByTestId('mainpageright')).toHaveClass(styles.expand);
```

Comment:
Why wasn't this file converted to a `.spec.` file?

Reply:
Its is a jest file not a vitest file